frontend/components/events/create/forms/BasicForm.tsx

```diff
@@ -254,28 +233,27 @@ export function BasicInformation({
                 size="lg"
               />
             </div>
-                </div>
-
-            <div>
-              <label className="text-black text-lg font-semibold">Is your event Private?</label>
-              <p className="text-sm mb-5 mt-2">
-                Private Events will not be shown on the public dashboard and will be invite only
-              </p>
-              <div className="mt-4">
-                <Select
-                  size="md"
-                  label="Select Visibility"
-                  value={isPrivate.toString()}
-                  onChange={(e) => {
-                    const privacyValue = e || "Public";
-                    handlePrivacyChange(privacyValue);
-                  }}
-                >
-                  <Option value="Public">Public</Option>
-                  <Option value="Private">Private</Option>
-                </Select>
-              </div>
+          </div>
+          <div>
```

Comment:
is this full width or the half width one?

Reply:
full width